README.md

```diff
@@ -4,6 +4,13 @@ This is 'perfgrind', tools for collecting samples from Linux performance events
 subsystem and converting profiling data to callgrind format, allowing it to be read with KCachegrind.
 
 Because of its own simplified format containing only the data necessary for creating the callgrind profile, the resulting file is commonly much smaller.
+One additional reason is that perfgrind explicit ignores the kernel space in its tracing.
```

Comment:
explicit**ly**?

in its tracing -> during profiling?

Reply:
taken